pgcli/main.py

```diff
@@ -1284,7 +1284,15 @@ def cli(
         try:
             cfg = load_config(pgclirc, config_full_path)
             dsn_config = cfg["alias_dsn"][dsn]
-        except:
+        except KeyError:
+            click.secho(
+                f"Could not find a DSN with alias {dsn}. "
```

Comment:
I used an f-string because I see you support Python3.6+, but I'm happy to switch to a different form of string interpolation if there's a preferred method for the package.

Reply:
We are very happy with f-strings now that we don't have to support python 2. :)